deployment/core-infrastructure/kustomize-templates/kuadrant/wasm-shim-config/kustomization.yaml

```
@@ -0,0 +1,18 @@
+# Patch the Kuadrant operator to use custom wasm-shim for token metrics
+patches:
+- target:
+    group: operators.coreos.com
+    version: v1alpha1
+    kind: ClusterServiceVersion
+    name: kuadrant-operator.v0.0.0
+    namespace: kuadrant-system
+  patch: |-
+    - op: replace
+      path: /spec/install/spec/deployments/0/spec/template/spec/containers/0/env/1/value
+      value: "oci://ghcr.io/nerdalert/wasm-shim:latest"
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)